tests/kda/test_recurrent_kda_prefill.py

```
@@ -4299,3 +4300,372 @@ def test_frozen_prefill_cuda_graph_workspaces_are_isolated(flash_kda_device):
+        kda_vibecuda_api._vibecuda_prefill_target(torch.device("cuda"))
+        == expected_target
+    )
+
+
+def test_vibecuda_prefill_target_rejects_unsupported_arch(monkeypatch):
+    monkeypatch.setattr(
+        kda_vibecuda_api,
+        "get_compute_capability",
+        lambda device: (9, 0),
+    )
+    with pytest.raises(RuntimeError, match="10.0 and 10.3"):
```

Ruff (0.16.2)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)